brave/brave-core
# Limit unblinded payment token redemption to a maximum of 50 per request [DO NOT MERGE]

Resolves https://github.com/brave/brave-browser/issues/25632

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm unblinded payment tokens are redeemed if there are less than 50
- Confirm unblinded payment tokens are redeemed in batches of 50 if there are 50 or more
- Confirm the correct unblinded payment tokens are removed from the `unblinded_payment_tokens` node in `confirmations.json`
